README.MD

```diff
@@ -71,44 +93,141 @@ This is the minimal required configuration. Take a look to the [Account Kit docu
           android:value="@string/ak_client_token" />
    </application>
    ...
-   ```
+```
 
-3. Check the `RNAccountKitPackage` is registered in your `MainApplication.java` file. This should be done by `react-native link` but sometimes it doesn't. You should see something like the following
-  ```java
+3.  Check the `RNAccountKitPackage` is registered in your `MainApplication.java` file. This should be done by `react-native link` but sometimes it doesn't. You should see something like the following
+
+```java
   @Override
   protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
           new MainReactPackage(),
           new RNAccountKitPackage() // <-- the package is registered
       );
   }
-  ```
+```
 
 This is the minimal required configuration. Take a look to the [Account Kit documentation for Android](https://developers.facebook.com/docs/accountkit/android) for a more detailed guide.
 
+</details>
+
+<details>
+    <summary>iOS</summary>
+
+- _A PR including iOS manual linking instructions is welcome_
+  </details>
+
+## Configuration
+
+<details>
+    <summary>iOS</summary>
+    <br/>
+
+Add your Facebook credentials to your project's `Info.plist` file
+
+```xml
+  <plist version="1.0">
+    <dict>
+      ...
+      <key>FacebookAppID</key>
+      <string>{your-app-id}</string>
+      <key>AccountKitClientToken</key>
+      <string>{your-account-kit-client-token}</string>
+      <key>CFBundleURLTypes</key>
+      <array>
+        <dict>
+          <key>CFBundleURLSchemes</key>
+          <array>
+            <string>ak{your-app-id}</string>
+          </array>
+        </dict>
+      </array>
+      ...
+    </dict>
+  </plist>
+```
+
+_This is the minimal required configuration. Take a look to the [Account Kit documentation for iOS](https://developers.facebook.com/docs/accountkit/ios) for a more detailed guide._
+
+</details>
+
+<details>
+    <summary>Android</summary>
+    <br/>
+
+1.  In `android/app/src/main/res/values/strings.xml`
+
+```xml
+...
+<string name="fb_app_id">YOUR_FACEBOOK_APP_ID</string>
+<string name="ak_client_token">YOUR_CLIENT_TOKEN</string>
```

Comment:
Don't  you have to add `app_name` here too?

Reply:
The example added in this PR `RN048_cocoapods` is currently not supporting Android.

Maybe adding Android support (which doesn't take much time) or simply just removing Android folder (since there is another example called `RN048` which does work on Android) is necessary

Reply:
The example added in this PR `RN048_cocoapods` is currently not supporting Android.

Maybe adding Android support (which doesn't take much time) or simply just removing Android folder (since there is another example called `RN048` which does work on Android) is necessary